allennlp/data/token_indexers/token_indexer.py

```diff
@@ -16,9 +17,20 @@ class TokenIndexer(Generic[TokenType], Registrable):
     34), or as lists of character IDs (e.g., "cat" gets represented by the numbers [23, 10, 18]),
     or in some other way that you can come up with (e.g., if you have some structured input you
     want to represent in a special way in your data arrays, you can do that here).
+
+    Parameters
+    ----------
+    token_min_padding_length : ``int``, optional (default=``0``)
+        The minimum padding length required for the :class:`TokenIndexer`. For example,
+        the minimum padding length of :class:``SingleIdTokenIndexer`` is the largest size of
+        filter when using :class:``CnnEncoder``.
```

Comment:
Maybe add the following:

```
Note that if you set this for one TokenIndexer, you likely have to set it for `all` TokenIndexers for the same field, otherwise you'll get mismatched tensor sizes.
```